app/controllers/pets_controller.rb

```diff
@@ -1,6 +1,12 @@
 class PetsController < ApplicationController
   def index
-    @pets = Pet.all
+    if params["adoptable"] == "true"
```

Comment:
not sure if this is standard for rails but it did allow to set up everything nicely in the view